src/plugins/plugins/permissions.js

```diff
@@ -131,4 +151,26 @@ export class Permissions extends ChatPlugin {
     const perms = Object.keys(this.bot.db.get(`permissions.groups.${group}`).value());
     return perms.join(', ');
   }
+
+  @help('/login userIdString userToken');
+  @permissionGroup('public');
+  @respond(/^login\s*(\S+)?\s*(\S+)?$/i);
+  async multipleAdapterLogin ([, userId, token], message) {
```

Comment:
The help text could be improved: `Use /login to receive a token to claim your user across chat adapters, or /login <userId> <userToken> to log in on another adapter.`